lib/seattleflu/id3c/cli/command/etl/redcap_det_uw_reopening.py

```diff
@@ -0,0 +1,1210 @@
+            is_complete('post_collection_data_entry_qc', redcap_record_instance)) or \
+            (collection_method == CollectionMethod.KIOSK and \
+            is_complete('kiosk_registration_4c7f', redcap_record_instance))
+
+        if specimen_received:
+            specimen_entry, specimen_reference = create_specimen(redcap_record_instance, patient_reference, collection_method)
+            specimen_observation_entry = create_specimen_observation_entry(
+                specimen_reference, patient_reference, initial_encounter_reference)
+        else:
+            LOG.info("Creating encounter for record instance without sample")
+
+        if specimen_received and not specimen_entry:
+            LOG.warning("Skipping record instance with insufficent information to construct a specimen")
+            continue
+
+        computed_questionnaire_entry = None
+        enrollment_questionnaire_entry = None
+        daily_questionnaire_entry = None
+        testing_determination_internal_questionnaire_entry = None
+
+        computed_questionnaire_entry = create_computed_questionnaire_response(
+            redcap_record_instance, patient_reference, initial_encounter_reference,
+            birthdate, datetime.strptime(initial_encounter_entry['resource']['period']['start'], '%Y-%m-%d'))
+
+        if event_type == EventType.ENROLLMENT:
+            enrollment_questionnaire_entry = create_enrollment_questionnaire_response(
+            enrollment, patient_reference, initial_encounter_reference)
+        else:
+            testing_determination_internal_questionnaire_entry = \
+                create_testing_determination_internal_questionnaire_response(
+                redcap_record_instance, patient_reference, initial_encounter_reference)
+
+            daily_questionnaire_entry = \
+                create_daily_questionnaire_response(
+                redcap_record_instance, patient_reference, initial_encounter_reference)
+
+        follow_up_encounter_entry = None
+        follow_up_questionnaire_entry = None
+        follow_up_computed_questionnaire_entry = None
+
+        if event_type == EventType.ENCOUNTER and is_complete('week_followup', redcap_record_instance):
+            follow_up_encounter_entry, follow_up_encounter_reference = create_follow_up_encounter(
+                redcap_record_instance, patient_reference, site_reference, initial_encounter_reference)
+            follow_up_questionnaire_entry = create_follow_up_questionnaire_response(
+            redcap_record_instance, patient_reference, follow_up_encounter_reference)
+            follow_up_computed_questionnaire_entry = create_computed_questionnaire_response(
+            redcap_record_instance, patient_reference, follow_up_encounter_reference,
+            birthdate, datetime.strptime(follow_up_encounter_entry['resource']['period']['start'], '%Y-%m-%d'))
+
+        current_entries = [
```

Comment:
The `resource_entries` / `current_entries` variable lists here are a little difficult for me to understand. Perhaps there's some clearer language we can use to distinguish between them? Assuming I have the right idea here, some suggestions that come to mind are `persistent_resource_entries` versus `instance_specific_entries`.

Reply:
OK. I switched to current_instance_entries and persisted_resource_entries.